infra/app/build-repository/main.tf

```diff
@@ -0,0 +1,51 @@
+locals {
+  project_name = "<PROJECT_NAME>"
+  app_name     = "<APP_NAME>"
+  region       = "<REGION>"
+
+  # Set project tags that will be used to tag all resources.
+  tags = {
+    project             = local.project_name
```

Comment:
There is already a project name defined in `infra/modules/common/outputs.tf  output "default_tags" {}`. Two thoughts/options related to this and one general one. 1. Remove project from "default_tags" or 2. Remove the `project = ` tag reference from here and then in  ` infra/modules/common/outputs.tf` create a new output “project_name” That you can use for project_name = module.common.project_name.